client/src/pages/Cars/index.tsx

```diff
@@ -1,8 +1,6 @@
-import CarsFilter from '../../components/CarsFilter';
-
 function Cars() {
   return (
-    <CarsFilter />
+    <h1>sd</h1>
```

Comment:
this folder should be the search page, SO use one of them either this or the search!